internal/services/controllers/jobs/controller.go

```diff
@@ -939,7 +953,7 @@ func (ec *JobsControllerImpl) handleStepRunFinished(ctx context.Context, task *m
 	}
 
 	// recheck the tenant queue
-	ec.checkTenantQueue(ctx, metadata.TenantId)
+	ec.checkTenantQueue(ctx, metadata.TenantId, "", false, true)
```

Comment:
empty string?

Reply:
yes, only matters for queueing, not for releasing a slot